weblate/trans/util.py

```diff
@@ -138,7 +138,7 @@ def get_clean_env(extra=None):
     }
     if extra is not None:
         environ.update(extra)
-    variables = ('PATH', 'LD_LIBRARY_PATH')
+    variables = ('PATH', 'LD_LIBRARY_PATH', 'SystemRoot')
```

Comment:
I'm not sure this should be done on all platforms, but I think this variable should be harmless elsewhere, so it's probably safe to whitelist it generally.